pkg/microservice/aslan/core/service/service/service.go

```diff
@@ -1362,6 +1365,16 @@ func getCronJobContainers(data string) ([]*commonmodels.Container, error) {
 	return containers, nil
 }
 
+func getImageName(image string) string {
```

Comment:
Provide an example?

Reply:
There are two places where this function is implemented and extracted to a common place?